Added support for virtualenv name in prompt

LuRsT opened the pull request:
This is how it looks like when it's enabled: ![](http://imgur.com/5lHCqzxl.png)

I chose that color for no reason in particular, seemed appropriate.

LuRsT committed:
Added support for virtualenv name in prompt

frmendes commented:
Hey! Thanks for the PR!

Just a couple of things:

* What do you think about changing the display to the right side of the prompt? Since we're putting most of the similar information on that end, it would be more consistent.
* Can you change the color to something like `green`? It would be more consistent with the rest of the theme regardless of what colorscheme the users activate.

LuRsT committed:
Moved virtualenv name to right and changed style

- From [] to ()
- Color changed to "green"

LuRsT commented:
Sounds sensible, I changed to normal green, replaced the square brackets with normal parenthesis (since it's the default way it appears in bash without any plugin) and moved it to the right. I get a bit of too much spacing when there's no git repo, but I'm not too worried about it.

![preview](http://imgur.com/X9Ld4NRl.png)

frm commented on geometry.zsh:
```diff
@@ -217,10 +226,10 @@ prompt_geometry_render() {
 
   PROMPT="
- %${#PROMPT_SYMBOL}{%(?.$GEOMETRY_PROMPT.$GEOMETRY_EXIT_VALUE)%} %F{$GEOMETRY_COLOR_DIR}%3~%f "
+%${#PROMPT_SYMBOL}{%(?.$GEOMETRY_PROMPT.$GEOMETRY_EXIT_VALUE)%} %F{$GEOMETRY_COLOR_DIR}%3~%f "
```
Can you add this extra space again?

frm commented on geometry.zsh:
```diff
@@ -217,10 +226,10 @@ prompt_geometry_render() {
 
   PROMPT2=" $GEOMETRY_SYMBOL_RPROMPT "
-  RPROMPT="$(prompt_geometry_git_info)%f"
+  RPROMPT="$(prompt_geometry_virtualenv) $(prompt_geometry_git_info)%f"
```
Can you move the space between both calls to the `prompt_geometry_virtualenv` `echo` call? The way you put it, it will display that extra space even if users have virtualenv disabled.

LuRsT commented:
Done!

frmendes commented:
Perfect! Thanks!

glhrmv commented:
Is this feature still working? I added the following line after `antigen theme geometry-zsh/geometry`:
```
GEOMETRY_RPROMPT+=(geometry_virtualenv)
```
And nothing happens when I `source bin/activate` a virtualenv.
